### PR TITLE
enable source file replacement in set_vm_disk method

### PR DESCRIPTION
**Pull request by chunfuwen**
Signed-off-by: chunfuwen <chwen@redhat.com>

### CODE REVIEW

**Comment by chunfuwen** [comment]
@kylazhang 
@balamuruhans 
please take a look

File: virttest/utils_test/libvirt.py [hunk 1]
```diff
@@ -2376,7 +2376,12 @@ def set_vm_disk(vm, params, tmp_dir=None, test=None):
         if disk_snap_name:
             disk_params_src.update({'source_snap_name': disk_snap_name})
     else:
-        # use current source file with update params
+        """
+        If disr_src_name is given, replace current source file
```
Comment (balamuruhans) [inline virttest/utils_test/libvirt.py]: %s/disr/disk

Comment (chunfuwen) [reply virttest/utils_test/libvirt.py]: done

**Comment by balamuruhans** [comment]
@chunfuwen, commit msg typo %s/disr_src_name/disk_src_name.

**Comment by balamuruhans** [comment]
@chunfuwen thanks, let us merge this PR.

**Pull request merged**